bitnami/argo-cd/templates/application-controller/_pod.tpl

```diff
@@ -87,11 +87,21 @@ initContainers:
           fi
     {{- if include "argocd.redis.auth.enabled" . }}
     env:
+      {{- if .Values.usePasswordFiles }}
+      - name: REDISCLI_AUTH_FILE
+        value: {{ printf "/opt/bitnami/argo-cd/secrets/%s" (include "argocd.redis.secretPasswordKey" .) }}
```

Comment:
Is this going to work? I don't see any `argo-cd-env.sh` script in the container image that auto-detects env. vars with `_FILE` suffix and extract the content:

- https://github.com/bitnami/containers/tree/main/bitnami/argo-cd/2/debian-12/rootfs/opt/bitnami/scripts/argo-cd

Reply:
Thanks! I was missing this in the container command:

```
          {{- if .Values.usePasswordFiles }}
            export REDISCLI_AUTH="$(< $REDISCLI_AUTH_FILE)"
          {{- end }}
```